### ISSUE TITLE
Flip ramp based on basemap?

### ISSUE THREAD

**Issue by makella** [CartoDB/deep-insights.js#350]
@fdansv related to this issue https://github.com/CartoDB/CartoColor/issues/2 and what we discussed this morning about my ramps being in backwards.

Is it possible in Auto Styling to read in the ramps as they are now for the dark basemap and then read them in the opposite way for the light?

The reason is that on the dark basemap, I think it would be good to have the lightest color = high where it stands out most and everything else is pushed back. On the other hand, on the light basemap, we could have the darkest color = high so it comes to the foreground.

**Comment by fdansv** [comment]
Will that be the general case, though? I can imagine that you wouldn't want that in a green-yellow-red situation, right?

**Comment by makella** [comment]
Right, so maybe the `A` ramp and the category ramps would stay unflipped. I think with the other ones we have for Auto Styling, that they will work ok like this. I don't believe this should be a default for everything. Is it possible for just auto styling?

**Comment by makella** [comment]
Oh which reminds me, we shouldn't flip the category ramps! Should I just redo the cartocolor file?

**Comment by fdansv** [comment]
you could indicate me which ones are 'flippable' and I'll flip'em!
![](http://www.liketotally80s.com/wp-content/uploads/2014/08/DevoFreedomofChoice.jpg)

**Comment by makella** [comment]
haha! ok! i'll add it to the issue there :)

**Comment by makella** [comment]
@fdansv now with the light/dark styling implemented, is it possible to have the ramps read in one way for light (dark color = high / light color = low) and the opposite for dark (dark color = low / light color = high)?
